public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -928,11 +976,20 @@ public SurveyQuestionResult apply(AchillesResult o) {
 
     @Override
     public ResponseEntity<EhrCountAnalysis> getEhrCountAnalysis(String domainId) {
-        CdrVersionContext.setCdrVersionNoCheckAuthDomain(defaultCdrVersionProvider.get());
+        try {
+            CdrVersionContext.setCdrVersionNoCheckAuthDomain(defaultCdrVersionProvider.get());
+        } catch(NullPointerException ie) {
+            throw new DataNotFoundException("Cdr Version table is not available. Please check if the database is up and version is right.");
+        }
         List<Long> analysisIds = new ArrayList<>();
         analysisIds.add(3300L);
         analysisIds.add(3301L);
         List<AchillesAnalysis> ehrAnalysesList = achillesAnalysisDao.findAnalysisByIds(analysisIds, domainId);
+
+        if (ehrAnalysesList == null) {
```

Comment:
Can never be null since 
`achillesAnalysisDao.findAnalysisByIds(analysisIds, domainId)` will never return null, but empty list instead.